run/sense2john.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+import xml.etree.ElementTree as ET
+
+def process_file(filename):
+    try:
+        fd = open(filename, "rb")
+    except IOError:
+        e = sys.exc_info()[1]
+        sys.stderr.write("%s\n" % str(e))
+        return
+
+    tree = ET.parse(filename)
```

Comment:
The script never uses `fd` obtained from `open`. Perhaps it'd be more appropriate to omit the `open` and to move the `ET.parse` inside a `try`.